src/test/java/org/springframework/data/tarantool/repository/SampleUserRepository.java

```diff
@@ -0,0 +1,13 @@
+package org.springframework.data.tarantool.repository;
+
+import org.springframework.data.tarantool.entities.SampleUser;
+
+
+/**
+ * @author Oleg Kuznetsov
+ */
+public interface SampleUserRepository extends TarantoolRepository<SampleUser, String> {
```

Comment:
Implicit methods probably must raise a sane exception here.
Add TCs for error cases

Reply:
What exception?

Reply:
@akudiyar it seems the comment is outdated. It was for the case when @Tuple is not present anywhere, so implicit methods will not work because they do not the actual space to take data from.